packages/react/src/analytics/integrations/Forter/types/globals.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Window {
+  jQuery: JQueryStatic;
```

Comment:
Is this doing anything?

Reply:
Yes, it was used on setupForterTokenLoadedEventListner to type window['jquery'] as JQueryStatic.

Reply:
Fixed by adding `@types/jquery` and changing `skipLibCheck` to false to allow type checking of `.d.ts` files